tests/end_to_end/serving.py

```diff
@@ -0,0 +1,48 @@
+import os
+import time
+
+from dotenv import load_dotenv
+
+import featureform as ff
+
+
+featureform_location = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+env_file_path = os.path.join(featureform_location, ".env")
+load_dotenv(env_file_path)
+
+VERSION=os.getenv("TEST_CASE_VERSION")
+
+SLEEP_DURATION = 30
+NUMBER_OF_SLEEPS = 20
+
+
+client = ff.ServingClient()
+
+def serve_data():
+    for _ in range(NUMBER_OF_SLEEPS):
+        try: 
+            dataset = client.training_set(f"fraud_training_{VERSION}", "quickstart")
+            training_dataset = dataset.repeat(10).shuffle(1000).batch(8)
+            for i, feature_batch in enumerate(training_dataset):
+                if i >= 1:
+                    return
+                print(feature_batch.to_list())
```

Comment:
see if there's any way to verify the contents of the serving, matching row num of the source set, etc, checking errors on serving iterator.